Makefile

```diff
@@ -50,32 +38,32 @@ endef
 
 
 # Makefile target
-### help : Show Makefile rules
+### help: Show Makefile rules
 .PHONY: help
 help:
 	@$(call func_echo_success_status, "Makefile rules:")
 	@echo
-	@awk $(ENV_HELP_AWK_RULE) Makefile
+	@grep -E '^### [-A-Za-z0-9_]+:' Makefile | sed 's/###/   /'
 	@echo
 
 
-### init_apisix : Fetch apisix code
+### init_apisix: Fetch apisix code
 .PHONY: init_apisix
 init_apisix:
 	@$(call func_echo_status, "$@ -> [ Start ]")
 	$(project_ci_runner) get_apisix_code
 	@$(call func_echo_success_status, "$@ -> [ Done ]")
 
 
-### patch_apisix : Patch apisix code
+### patch_apisix: Patch apisix code
 .PHONY: patch_apisix
 patch_apisix:
 	@$(call func_echo_status, "$@ -> [ Start ]")
 	$(project_ci_runner) patch_apisix_code
 	@$(call func_echo_success_status, "$@ -> [ Done ]")
 
 
-### install : Install custom plugin
+### install: Install custom plugin
```

Comment:
Not related to the current PR?

Reply:
The original help output has bug, using wrong awk codes:

```
[ info ] Makefile rules:

awk: line 1: syntax error at or near ,
awk: line 1: syntax error at or near }
make: *** [Makefile.orig:58: help] Error 2
```

I change it to common help output from other api7 projects. It's just a trivial but necessary bugfix.